.experimenter.json

```diff
@@ -1,12 +1,20 @@
 {
-  "search-term-groups": {
-    "description": "A feature allowing the grouping of URLs around the search term that it came from.",
+  "nimbus-validation": {
```

Comment:
maybe this could be commit in separate PR to just have what it's related to this PR :)

Reply:
i legitimately don't know how this gets sneaky in, i'm not trying to change this file lmao. i've done a few rebases and it keeps showing up from some commit

Reply:
Removed those changes